build_scripts/consul_setup/config.json

```diff
@@ -0,0 +1,16 @@
+{
+    "bootstrap": true,
+    "server": true,
+    "ui": true,
+    "node_name": "openrecords_consul",
+    "bind_addr": "10.0.0.5",
+    "data_dir": "/var/consul",
+    "log_level": "INFO",
+    "enable_syslog": true,
+    "encrypt": "pmsKacTdVOb4x8/Vtr9PWw==",
```

Comment:
Can we make this key say something like `CHANGE THIS BEFORE DEPLOYING TO PRODUCTION`

I don't want to forget..... 😅